modules/gltf/gltf_state.cpp

```diff
@@ -335,6 +336,15 @@ Node *GLTFState::get_scene_node(GLTFNodeIndex idx) {
 	return scene_nodes[idx];
 }
 
+GLTFNodeIndex GLTFState::get_node_index(Node *p_node) {
```

Comment:
In your experience is this slow? It's really iterating but I don't think it'll be slow for a 1000 nodes.

Reply:
I only tested it on small scenes and I did not benchmark it, but I would imagine it's not slow.